gnd/src/main/java/com/google/android/gnd/persistence/local/room/converter/ResponseJsonConverter.java

```diff
@@ -66,6 +67,6 @@ private static Object toJsonArray(MultipleChoiceResponse response) {
         Timber.e("Error parsing JSONArray in db: %s", jsonArray);
       }
     }
-    return list;
+    return ImmutableList.copyOf(list);
```

Comment:
Consider using ImmutableList.builder() instead of copying the list at the end.

Reply:
Done